sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -187,6 +195,11 @@ private Worker(
       this.exporterTimeoutMillis = exporterTimeoutMillis;
       this.queue = queue;
       this.batch = new ArrayList<>(this.maxExportBatchSize);
+      spansInQueue.setCallback(
+          result ->
+              result.observe(
+                  queue.size(),
+                  Labels.of("spanProcessorType", BatchSpanProcessor.class.getSimpleName())));
```

Comment:
This makes me think that we should have both the class name and the key be static fields of the Worker class, since we use them quite a bit. What do you think about making that simple refactoring?

Reply:
+1 I'll do the change.

Reply:
Done.